File: schemas/test_schema.py
from pydantic import BaseModel
from typing import Optional, List

class DifferentTestSchema(BaseModel):
    test: str

    class Config:
        title = "DifferentTestSchema"

class TestSchema(BaseModel):
    string_test: str
    int_test: int
    float_test: float
    bool_test: bool
    optional_test: Optional[DifferentTestSchema] = None
    test_list: List[str]

    class Config:
        title = "TestSchema"

File: klaxonModels/TestSchema.kt
package io.package.package.schemas

data class TestSchema(
	val string_test: String,
	val int_test: Integer,
	val float_test: Float,
	val bool_test: Boolean,
	val optional_test: DifferentTestSchema?,
	val test_list: List<String>,
	val test_list_2: List<DifferentTestSchema>
)